### PR TITLE
ci(cla-check): add CLA check for ubuntu-24.04

### PR DESCRIPTION
**Pull request by cjdcordeiro**
# Proposed changes
Add the CLA checks for the 24.04 branch.

## Testing
Worked on #132 - https://github.com/canonical/chisel-releases/actions/runs/8083045516

## Checklist

* [x] I have read the [contributing guidelines](
https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md)
* [x] I have already submitted the [CLA form](
https://ubuntu.com/legal/contributors/agreement)

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -10,6 +10,11 @@ on:
       - "ubuntu-*"
 
 jobs:
+  cla-check:
+    if: github.event_name == 'pull_request'
```
Comment (rebornplusplus) [inline .github/workflows/ci.yaml:14]: I am wondering if we should have added this condition inside the reusable workflow in `main`. It's no biggie though, so let's just go with it.

**Review by rebornplusplus — APPROVED** [review]
Looks good to me, thank you!

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (cjdcordeiro) [reply .github/workflows/ci.yaml:14]: hum could be indeed. although, wouldn't it be `workflow_call` then?

**Review by linostar — APPROVED** [review]
LGTM

**Pull request merged**